bridge/slack/helpers_test.go

```diff
@@ -0,0 +1,28 @@
+package bslack
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+)
+
+func TestExtractTopicOrPurpose(t *testing.T) {
+	testcases := []struct {
+		input          string
+		wantChangeType string
+		wantOutput     string
+	}{
+		{"@someone set channel topic: one liner", "topic", "one liner"},
+		{"@someone set channel purpose: one liner", "purpose", "one liner"},
+		{"@someone set channel topic: multi\nliner", "topic", "multi\nliner"},
+		{"@someone cleared channel topic", "topic", ""},
+		{"some unmatched message", "unknown", ""},
+	}
+
+	for _, tc := range testcases {
+		gotChangeType, gotOutput := extractTopicOrPurpose(tc.input)
+
+		assert.Equal(t, tc.wantChangeType, gotChangeType)
+		assert.Equal(t, tc.wantOutput, gotOutput)
```

Comment:
It's quite handy to use the `assert.Equalf` variants here and provide an error message that tells (together with my comment above) the name of the testcase where the assertion failed.

Reply:
Interestingly, Equal and Equalf seem to be funcationally equivalent... But I like using Equalf by convention anyhow